src/endpoints/google.js

```diff
@@ -46,13 +46,13 @@ router.post('/caption-image', jsonParser, async (request, response) => {
 
         if (!result.ok) {
             const error = await result.json();
-            console.log(`Google AI Studio API returned error: ${result.status} ${result.statusText}`, error);
+            console.error(`Google AI Studio API returned error: ${result.status} ${result.statusText}`, error);
             return response.status(result.status).send({ error: true });
         }
 
         /** @type {any} */
         const data = await result.json();
-        console.log('Multimodal captioning response', data);
+        console.debug('Multimodal captioning response', data);
```

Comment:
Another one that I would change to info. I can see why having the data logged would be more of a debug part - but we should really log when calls to an external API are happening, etc.